src/Views/Form/Themes/Sequoia/Actions.php

```diff
@@ -3,6 +3,7 @@
 namespace Give\Views\Form\Themes\Sequoia;
 
 use Give_Donate_Form;
+use function Give\Helpers\Form\Theme\get;
```

Comment:
Please use an alias `Theme\get as getTheme`, as `get` is way too vague.

Reply:
fixed: https://github.com/impress-org/givewp/compare/965ab69fb3f6...fb342b64d1a7